com.github.akiraux.libgtkcanvas.json

```diff
@@ -0,0 +1,35 @@
+{
+   "app-id": "com.github.akiraux.libgtkcanvas",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "3.30",
+   "sdk": "org.gnome.Sdk",
+   "base":"io.elementary.BaseApp",
+   "base-version": "juno",
+   "command": "gtkcanvas-demo",
+   "cleanup": [
+      "/include",
+      "/lib/pkgconfig",
+      "/lib/debug",
+      "/share/vala",
+      "/man",
+      "*.a",
+      "*.la"
+   ],
+   "finish-args": [
+      "--share=ipc",
+      "--socket=wayland",
+      "--share=network"
```

Comment:
```suggestion
```
I don't think we need network here ^^ we might need to add x11 too in case someone tries it on x11.